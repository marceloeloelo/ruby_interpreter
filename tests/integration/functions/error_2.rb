def method_1(a, b)
end

method_1(1) # ArgumentError: wrong number of arguments (2 for 1)
