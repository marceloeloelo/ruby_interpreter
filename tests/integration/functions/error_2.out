ArgumentError: wrong number of arguments (2 for 1)
